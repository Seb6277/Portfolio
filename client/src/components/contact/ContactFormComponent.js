import React from 'react'
import {Button, FormGroup, Form, Label, Input} from 'reactstrap'

class ContactFormComponent extends React.Component{
  render() {
    return(
      <Form className="contact_form col-md-10 offset-md-1 col-sm-10 offset-sm-1">
        <FormGroup>
          <Label for="email">E-mail :</Label>
          <Input type="email" name="email" id="email"/>
        </FormGroup>
        <FormGroup>
          <Label for="text">Message :</Label>
          <Input type="textarea" name="text" id="text" />
        </FormGroup>
        <Button>Envoyer</Button>
      </Form>
    )
  }
}

export default ContactFormComponent