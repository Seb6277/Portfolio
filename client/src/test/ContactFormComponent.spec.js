import ContactFormComponent from '../components/contact/ContactFormComponent'

describe('<ContactFormComponent/>', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<ContactFormComponent/>)
    expect(wrapper).toMatchSnapshot()
  });
})