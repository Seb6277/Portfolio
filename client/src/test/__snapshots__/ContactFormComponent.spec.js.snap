// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactFormComponent/> should render correctly 1`] = `
<Form
  className="contact_form col-md-10 offset-md-1 col-sm-10 offset-sm-1"
  tag="form"
>
  <FormGroup
    tag="div"
  >
    <Label
      for="email"
      tag="label"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      E-mail :
    </Label>
    <Input
      id="email"
      name="email"
      onChange={[Function]}
      type="email"
      valid={false}
    />
  </FormGroup>
  <FormGroup
    tag="div"
  >
    <Label
      for="text"
      tag="label"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      Message :
    </Label>
    <Input
      id="text"
      name="text"
      onChange={[Function]}
      type="textarea"
      valid={false}
    />
  </FormGroup>
  <div
    className="captcha col-md-6 offset-md-2"
  >
    <AsyncScriptLoader(ReCAPTCHA)
      onChange={[Function]}
      sitekey="6LdIZqkUAAAAAO0mwtqMNSfGHdUjna0eFmFRrbg9"
    />
  </div>
  <Button
    color="secondary"
    disabled={true}
    onClick={[Function]}
    tag="button"
  >
    Envoyer
  </Button>
</Form>
`;
